Gems/Atom/Feature/Common/Code/Source/PostProcess/DepthOfField/DepthOfFieldSettings.cpp

```diff
@@ -159,8 +160,25 @@ namespace AZ
             float focusDistance = 0.0f;
             if (m_enableAutoFocus)
             {
-                focusDistance = m_viewNear + m_normalizedFocusDistanceForAutoFocus * (m_viewFar - m_viewNear);
-                focusDistance = GetClamp(focusDistance, m_viewNear, m_viewFar);
+                if (m_focusedEntityId.IsValid())
+                {
+                    AZ::Vector3 focusPosition;
+                    AZ::TransformBus::EventResult(focusPosition, m_focusedEntityId, &AZ::TransformBus::Events::GetWorldTranslation);
+
+                    AZ::Transform cameraTransform;
+                    AZ::TransformBus::EventResult(cameraTransform, m_cameraEntityId, &AZ::TransformBus::Events::GetWorldTM);
+
+                    AZ::Vector3 cameraPosition = cameraTransform.GetTranslation();
```

Comment:
You could get the `cameraPosition` directly by using `GetWorldTranslation` event as for `focusPosition`

Reply:
I later use the transform to get the camera's `forward` direction, so I need to get the transform anyway.